website/docs/docs/quickstarts/dbt-cloud/redshift-qs.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "Quickstart for dbt Cloud and Redshift"
+description: "Quickstart for dbt Cloud and Redshift."
+id: "redshift"
+sidebar_label: "Redshift quickstart"
+---
+For the Redshift project in the quickstart guide, you'll learn how to:
+
+- Set up a Redshift cluster.
+- Load sample data into your Redshift account.
+- Connect dbt Cloud to Redshift.
+- Take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.
+- Add tests to your models
+- Document your models
+- Schedule a job to run
+
+
+:::tip Videos for you
+You can check out [dbt Fundamentals](https://courses.getdbt.com/courses/fundamentals) for free if you're interested in course learning with videos.
+
+:::
+
+## Prerequisites 
+- You have a  [dbt Cloud account](https://cloud.getdbt.com/). 
+- You have an AWS account with permissions to execute a CloudFormation template to create appropriate roles and a Redshift cluster.
+
+## Create a Redshift cluster
+1. Sign in to your [AWS account](https://signin.aws.amazon.com/console) as a root user or an IAM user depending on your level of access.
+2. Use a CloudFormation template to quickly set up a Redshift cluster. A CloudFormation template is a configuration file that automatically spins up the necessary resources in AWS. [Start a CloudFormation stack](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=dbt-workshop&templateURL=https://tpch-sample-data.s3.amazonaws.com/create-dbtworkshop-infr) and you can refer to the [create-dbtworkshop-infr JSON file](https://github.com/aws-samples/aws-modernization-with-dbtlabs/blob/main/resources/cloudformation/create-dbtworkshop-infr) for more template details.
```

Comment:
This portion could use some work. For a workshop, these instructions are fine, but this is presumably so we can get them to deployment, and we're asking them to blindly accept a number of pre-configured fields without explaining what any of them are doing. Many of these fields are also filled in specifically for a workshop. We should work with an SME to get a JSON that's built for broader purposes. 
The README file in the Github repo only says that it's for workshop use only; this could use some fleshing out.

Reply:
great feedback! for the purposes of this quickstart, i think it's a bit out of scope. 

how should we capture this tho? maybe this can be an improvement project for all 4 quickstarts?